torchreid/data/datamanager.py

```diff
@@ -236,13 +236,13 @@ def __init__(
         assert isinstance(self._num_train_pids, list)
         assert isinstance(self._num_train_cams, list)
         assert len(self._num_train_pids) == len(self._num_train_cams)
-
+        batch_size_train = self.calculate_batch(batch_size_train, len(train_dataset))
```

Comment:
I'd rather disable that by default since it is useful only for certain datasets and training configs.

Reply:
Do you propose to include a condition and add a new parameter to the config file?

Reply:
Yes, let's add it